packages/p2p-v2/src/pages/my-ads/components/BuyAdPaymentSelection/BuyAdPaymentSelection.scss

```diff
@@ -0,0 +1,17 @@
+.p2p-v2-buy-ad-payment-selection {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0.5rem 1.2rem 0;
+    border-radius: 4px;
+    border: 1px solid #d6dadb;
+    margin-bottom: 2rem;
+
+    & .deriv-button {
+        &__variant--contained {
+            &.deriv-button__color--white:hover:not(:disabled) {
+                background-color: transparent;
+            }
+        }
+    }
+}
```

Comment:
can we not just use the className below you added for the button, i know there are current issues with the button hovering styles, we could just use !important for now, and remove it once they have implemented the changes for removing hover styles

Reply:
better add a comment to remove this once the hover styles removal is added